newIDE/app/src/EventsSheet/index.js

```diff
@@ -562,10 +566,31 @@ export default class EventsSheet extends React.Component<Props, State> {
         ? parameterContext.domEvent.currentTarget
         : null,
       inlineEditingChangesMade: false,
+      inlineEditingPreviousValue: instruction.getParameter(parameterIndex),
     });
   };
 
-  closeParameterEditor = () => {
+  closeParameterEditor = (
+    event: SyntheticKeyboardEvent<HTMLInputElement>,
+    peferenceEventsSheetCancelInputParameterPopover: boolean
```

Comment:
Use a shorter name for your boolean, we don't care here that it comes from the preferences.